content/applications/websites/ecommerce/ecommerce_management/customer_accounts.rst

```diff
@@ -0,0 +1,47 @@
+=================
+Customer accounts
+=================
+
+Having customer accounts on an eCommerce allows customers to access all their documents in a single
+place. To access their account, customers must be **logged-in** on the eCommerce website and under
+their **account name**, click :guilabel:`My Account`. From there, customers have access to their
+:guilabel:`quotations`, :guilabel:`orders`, :guilabel:`invoices`, etc.
+
+.. image:: customer_accounts/account-log.png
+   :align: center
+   :alt: Customer account log-in
+
+.. note::
+   Customers can only have an account if the option is :ref:`enabled <checkout-sign>`.
+
+.. tip::
+   Similarly to the rest of the website, the customer account page can be customized with **content
+   blocks** and other features through the **website builder**.
+
+Access restriction
+==================
+
+It is possible to allow or restrict the documents to which customers have access through the website
+builder. Log in your **own** account with your Odoo database credentials, and go to
+:menuselection:`Edit --> Customize`. From the website builder menu, enable or disable the documents
+customers can have access to.
+
+.. image:: customer_accounts/account-documents.png
+   :align: center
+   :alt: Documents to which customers have access to from their account
+
+Account security
+================
+
+Customers can edit their password by clicking :guilabel:`Edit Security Settings` on the homepage of
+their account. Additionally, if the option :guilabel:`Enforce two-factor authentication` (under the
+**General Settings** app) is enabled for :guilabel:`All users`, customers can add a
+second-authentication step. For that, they must click :guilabel:`Enable two-factor authentication`
+in the :guilabel:`Edit Security Settings`, and setup an **Authenticator app** of their choice.
+
+Multi-website account
+=====================
+
+If you own multiple websites, customer accounts can be made usable across **all** websites. The
+customer then only needs one account. To do so, go to :menuselection:`Website --> Configuration -->
+Settings --> Privacy section`, and enable :guilabel:`Shared Customer Accounts`.
```

Comment:
If you own multiple websites, customer accounts can be usable across **all** websites. Then, the
customer only needs one account.